style.css

```diff
@@ -0,0 +1,251 @@
+@import url('https://fonts.googleapis.com/css2?family=Dosis&display=swap');
+
+body {
+    margin: 0;
+}
+
+
+.main {
+    display: flex;
+    flex-direction: column;
+    font-family: Dosis;
+}
+
+
+
+.header {
+    background-image: url(./image/bg\ header.png);
+    width: 100%;
+    height: 626px;
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    background-size: cover;
+    width: 100%;
+    max-width: 1920px;
+}
+
+.watch200 {
+    top: 256px;
+    margin-left: 69px;
+}
+
+.horse {
+    margin-top: -43px;
+    margin-left: 110px;
+}
+
+.brain {
+    margin-top: 116px;
+    margin-left: -281px;
+}
+
+.header-section {
+    display: flex;
+    margin: 0 auto;
+}
+
+.icon-section {
+    display: flex;
+    position: relative;
+}
+
+.txt-section {
+    display: flex;
+    flex-direction: column;
+}
+
+.navigation {
+    display: flex;
+    width: 90%;
+    height: 171px;
+    justify-content: center;
+    gap: 200px;
+    margin-top: 19px;
+}
+
+.nav-menu {
+    display: flex;
+    color: white;
+    gap: 100px;
+    margin-top: 22px;
+}
+
+.nav-menu a {
+    color: white;
+    text-decoration: none;
+}
+
+.about {
+    margin-top: 144px;
+    font-size: 100px;
+}
+
+
+.main-tags {
+    display: flex;
+    margin-top: 74px;
+    gap: 10px;
+    margin-left: 360px;
```

Comment:
remove `margin`, do it on another way, with `max-width `or just center, but without `margin`